### DISCUSSION TITLE
saleor as marketplace, multivendor like ebay, Amazon

### DISCUSSION THREAD

**Discussion by aksthelion** [saleor/saleor#11135] [Deployment]
Hello, I just come across your framework and it looks promising. I wanted to know if we can customise this framework to use as a marketplace allowing multiple sellers to sell their products. In other words I Want to build a platform like ebay letting anyone to sell or buy products. I've read the other question posted on this community regarding this but I didn't get a clear answer. Any inputs will be appreciated. Thanks.

**Comment by benjamin-carriou** [comment]
Hi,

You can use channels to seperate sellers products. 
One channel per seller => https://docs.saleor.io/docs/3.x/developer/channels

**Comment by karolkielecki** [comment]
Hi @aksthelion, thanks for asking. Saleor doesn't support the multi-vendor feature out of the box. I haven't seen any use case where someone built a marketplace like eBay on top of Saleor. You would need to modify the code heavily. 
There is an integration with Onport - https://onport.com/partners/store/saleor that supports some marketplace functionalities. 

The multi-channel mentioned by @benjamin-carriou allows you to manage multiple sales channels from a single dashboard.
